Charts/Core/vtkChartXY.cxx

```diff
@@ -1608,6 +1631,69 @@ bool vtkChartXY::MouseMoveEvent(const vtkContextMouseEvent& mouse)
       this->Scene->SetDirty(true);
     }
   }
+  else if (mouse.GetButton() == this->Actions.ClickAndDrag())
+  {
+    if (this->DragPoint)
+    {
+      // Iterate through each corner, and check for a nearby point
+      for (size_t i = 0; i < this->ChartPrivate->PlotCorners.size(); ++i)
+      {
+        int items = static_cast<int>(this->ChartPrivate->PlotCorners[i]
+                                     ->GetNumberOfItems());
+        if (items)
```

Comment:
Just to avoid some piece of code inside of another if, it will be better if you change that to
```C++
if (!items){
    continue;
}
```